src/components/FilterDrawer.js

```diff
@@ -0,0 +1,219 @@
+                        <FilterCheckboxes
+                            filter={carnivoreFilter}
+                            allLabel="All Carnivores"
+                            updateValues={this.updateFilter('carnivoreFilter')}
+                            briefNumber={Object.keys(carnivoreFilter).length - 1}
+                            keyColorFunction={getColorForSpecies}/>
+                    )}
+                    <hr/>
+
+                    {/* Neighborhoods */}
+                    {this.getCollapse(classes, "Neighborhood", this.toggleShow('showNeighborhoods'), showNeighborhoods,
+                        <FilterCheckboxes
+                            filter={neighborhoodFilter}
+                            allLabel="All Neighborhoods"
+                            updateValues={this.updateFilter('neighborhoodFilter')}
+                            briefNumber={briefNeighborhoodsCount}/>
+                    )}
+                    <hr/>
+
+                    {/* Time and Day */}
+                    {this.getCollapse(classes, "Time of Sighting", this.toggleShow('showTime'), showTime,
+                        <>
+                            Date:
+                            <DatePicker
+                                value={date}
+                                onChange={this.handleDateChange}/> <br/>
+                            Time of Day:
+                            <FilterCheckboxes
+                                filter={timeFilter}
+                                allLabel="Any time of day"
+                                updateValues={this.updateFilter('timeFilter')}
+                                briefNumber={Object.keys(timeFilter).length - 1}/>
+                        </>
+                    )}
+                    <hr/>
+
+                    {/* Confidence */}
+                    {this.getCollapse(classes, "Confidence of Sighting", this.toggleShow('showConfidence'), showConfidence,
+                        <FormControl component="fieldset">
+                            <FormGroup>
+                                <FormControlLabel
+                                    control={<Checkbox
+                                                checked={confidenceFilterActive}
+                                                onChange={this.toggleConfidence}
+                                                checkedIcon={<CheckBoxIntermediateIcon style={{color: '#93C838'}}/>}
+                                                style={{margin: '0px 0px 0px 8px'}}/>}
+                                    label="Only show high-confidence sightings" />
+                            </FormGroup>
+                        </FormControl>
+                    )}
+                </div>
+                <Button variant="contained"
+                    className={classes.resultsButton}
+                    onClick={() => console.log({date, confidenceFilterActive, carnivoreFilter, neighborhoodFilter, timeFilter})}>
```

Comment:
I'm actually not sure about this button in general -- it's not actually doing anything on desktop and there are two other ways to go to the map on mobile. Do you think we need it? I think it would be easy to update onclick to close it on mobile if that's the direction we want to go.

And we certainly don't need to log this, good call.